glide.lock

```diff
@@ -305,7 +305,7 @@ imports:
 - name: github.com/dgrijalva/jwt-go
   version: 01aeca54ebda6e0fbfafd0a524d234159c05ec20
 - name: github.com/docker/distribution
-  version: 114d1c11fb9258426f7197995cf4a3dfe46ef472
+  version: 9e3018b6ff58e5b86c76b5f9b8826494caca28ea
```

Comment:
didn't expect this...

Reply:
looks like the bot is still pushing to the docker/distribution branch from origin. we should turn that off for the forks we don't want it to maintain

Reply:
> looks like the bot is still pushing to the docker/distribution branch from origin. we should turn that off for the forks we don't want it to maintain

@mfojtik